src/main/java/io/github/thebusybiscuit/slimefun4/utils/ArmorStandUtils.java

```diff
@@ -0,0 +1,56 @@
+package io.github.thebusybiscuit.slimefun4.utils;
+
+import javax.annotation.Nonnull;
+
+import org.bukkit.Location;
+import org.bukkit.entity.ArmorStand;
+
+import io.github.thebusybiscuit.slimefun4.core.services.holograms.HologramsService;
+import io.github.thebusybiscuit.slimefun4.implementation.items.altar.AncientPedestal;
+import io.github.thebusybiscuit.slimefun4.implementation.items.blocks.HologramProjector;
+
+/**
+ * This class holds utilities for {@link ArmorStand}, useful for classes
+ * dealing with {@link ArmorStand}s that are not from {@link HologramsService}
+ *
+ * @see HologramProjector
+ * @see AncientPedestal
+ *
+ * @author JustAHuman
+ */
+public class ArmorStandUtils {
+    /**
+     * Spawns an {@link ArmorStand} at the given {@link Location} with the given custom name
+     * (Invisible, Silent, Marker, NoGravity, NoBasePlate, NoRemoveWhenFarAway, VisibleCustomName)
```

Comment:
? what is this list for?

Reply:
Just to tell who ever is using the util the settings the armorstand has

Reply:
If we want to list these I'd make it a proper message. `Set properties: <list>`
This will also just be part of the first line, I think you're wanting a line break here (`<br />`)